mocks/mocks.py

```diff
@@ -84,11 +75,18 @@ def scale_light_by_distance(particle_positions,
     """
     # Computing the length value that should be added to the positions of
     # particles in the viewing axis (projection axis) as correction.
-    physical_distance = ensure_unit(physical_distance, u.Mpc)
+    config['galaxy_distance'] = kwargs.get('galaxy_distance',
+                                           config['galaxy_distance'])
+    config['galaxy_distance'] = ensure_unit(config['galaxy_distance'], u.Mpc)
+    config['target_galaxy_comoving_depth'] =\
+        kwargs.get('target_galaxy_comoving_depth',
+                   config['target_galaxy_comoving_depth'])
+    config['target_galaxy_comoving_depth'] = ensure_unit(
+        config['target_galaxy_comoving_depth'], u.Mpc)
```

Comment:
Don't do all this stuff in this function, it doesn't belong here. Do it in a `parse_config` function.

Reply:
@AnthonyHorton 
Yes, I will. I just referenced your previous comment in another issue to start creating some functions, including `parse_config`, in another PR.

Reply:
This is just a clue for me for creating the `parse_config` function.

Reply:
Why not just put it in the right place to begin with? It not off topic for this PR. Seems to just be creating more work to write the code in one place, then immediately move it somewhere else. You'll have to modify a bunch of tests twice if you do that.

Reply:
This is all redundant now that it's being done in `parse_config`. You should remove it from here.